sql/src/main/java/io/crate/operation/collect/stats/StatsTablesService.java

```diff
@@ -133,6 +133,15 @@ private void setJobsLogSink(int size, TimeValue expiration) {
         statsTables.updateJobsLog(jobsLogSink);
     }
 
+    static long clearInterval(long expiration) {
```

Comment:
please add javadocs do explain what this method does. It's not obvious from the name. Also document the unit for expiration.

Could also consider using a TimeValue so that the unit is clear. (sinks are not changed often, so the extra allocation won't hurt us)